client/state/sites/test/selectors.js

```diff
@@ -1184,4 +1228,778 @@ describe( 'selectors', () => {
+					URL: 'https://jetpacksite.me',
+					jetpack: true,
+					options: {
+						active_modules: [ 'manage', 'sso', 'photon', 'omnisearch' ]
+					}
+				}
+			} );
+
+			const modulesActive = verifyJetpackModulesActive( state, siteId, [ 'after-the-deadline', 'manage' ] );
+			expect( modulesActive ).to.equal( false );
+		} );
+	} );
+
+	describe( '#getJetpackSiteRemoteManagementURL()', () => {
+		it( 'should return `null` for a non-existing site', () => {
+			const managementUrl = getJetpackSiteRemoteManagementURL( stateWithNoItems, nonExistingSiteId );
+			expect( managementUrl ).to.equal( null );
+		} );
+
+		it( 'it should return `false` for a non jetpack site', () => {
+			const state = createStateWithItems( {
+				[ siteId ]: {
+					ID: siteId,
+					options: {}
+				}
+			} );
+
+			const managementUrl = getJetpackSiteRemoteManagementURL( state, siteId );
+			expect( managementUrl ).to.equal( null );
+		} );
+
+		it( 'it should return the correct url for version of jetpack less than 3.4', () => {
+			const state = createStateWithItems( {
+				[ siteId ]: {
+					ID: siteId,
+					URL: 'https://jetpacksite.me',
+					jetpack: true,
+					options: {
+						active_modules: [ 'manage', 'sso', 'photon', 'omnisearch' ],
+						admin_url: 'https://jetpacksite.me/wp-admin/',
+						jetpack_version: '3.3'
+					}
+				}
+			} );
+
+			const managementUrl = getJetpackSiteRemoteManagementURL( state, siteId );
+			expect( managementUrl ).to.equal( 'https://jetpacksite.me/wp-admin/admin.php?page=jetpack&configure=manage' );
+		} );
+
+		it( 'it should return the correct url for versions of jetpack greater than or equal to 3.4', () => {
+			const state = createStateWithItems( {
+				[ siteId ]: {
+					ID: siteId,
+					URL: 'https://jetpacksite.me',
+					jetpack: true,
+					options: {
+						active_modules: [ 'manage', 'sso', 'photon', 'omnisearch' ],
+						admin_url: 'https://jetpacksite.me/wp-admin/',
+						jetpack_version: '3.4'
+					}
+				}
+			} );
+
+			const managementUrl = getJetpackSiteRemoteManagementURL( state, siteId );
+			expect( managementUrl ).to.equal( 'https://jetpacksite.me/wp-admin/admin.php?page=jetpack&configure=json-api' );
+		} );
+	} );
+
+	describe( '#hasJetpackSiteCustomDomain()', () => {
+		it( 'should return `null` for a non-existing site', () => {
+			const hasCustomDomain = hasJetpackSiteCustomDomain( stateWithNoItems, nonExistingSiteId );
+			expect( hasCustomDomain ).to.equal( null );
+		} );
+
+		it( 'it should return `true` if `URL` and `unmapped_url` have the same domain', () => {
+			const state = createStateWithItems( {
+				[ siteId ]: {
+					ID: siteId,
+					URL: 'https://jetpacksite.me',
+					options: {
+						unmapped_url: 'https://jetpack.co'
+					}
+				}
+			} );
+
+			const hasCustomDomain = hasJetpackSiteCustomDomain( state, siteId );
+			expect( hasCustomDomain ).to.equal( true );
+		} );
+
+		it( 'it should return `false` if `URL` and `unmapped_url` have different domains', () => {
```

Comment:
`unmapped_url` and `URL` use the same domains. Most likely test description needs an update ;)